cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -81,6 +90,19 @@ func runCreateManagedJob(cmd *cobra.Command, args []string) (err error) {
 	if err != nil {
 		return err
 	}
+	if options.manager {
+		if strings.Contains(bpCluster.ClusterURL, "api.stage") {
```

Comment:
This is only relevant if the cluster is a normal OSD/ROSA cluster - for Hypershift clusters this should run just fine in staging.